src/main/java/life/qbic/util/ProgressBar.java

```diff
@@ -44,19 +71,13 @@ private String shortenFileName(String fullFileName) {
     return shortName;
   }
 
-  private void checkForJump() {
-    if (this.downloadedSize > this.nextProgressJump) {
-      this.nextProgressJump += this.stepSize;
-      drawProgress();
-    }
-  }
-
-  public void draw() {
-    drawProgress();
+  private void drawProgress() {
+    System.out.printf("\r%-" + computeLeftPadding() + "s %s", this.fileName, buildProgressBar());
+    lastUpdated = System.currentTimeMillis();
   }
 
-  private void drawProgress() {
-    System.out.printf("%-" + computeLeftPadding() + "s %s\r", this.fileName, buildProgressBar());
+  public void remove() {
+    System.out.print("\r\\033[2K"); //clear whole line
```

Comment:
I don't think this function works as intended (See beginning of line):
<img width="1188" alt="Bildschirmfoto 2023-02-20 um 12 27 34" src="https://user-images.githubusercontent.com/29627977/220094027-bff842f2-0273-4e9c-8c66-e9521e621909.png">
Also the method name could be more explicit like clearPrintLine() or simliar 👍